cocos/scripting/lua-bindings/manual/CCLuaEngine.cpp

```diff
@@ -36,7 +36,13 @@
 #include "base/CCDirector.h"
 #include "base/CCEventCustom.h"
 
-#pragma comment(lib,"lua51.lib")
+#if (CC_TARGET_PLATFORM == CC_PLATFORM_OHOS)
+    #if _MSC_VER
```

Comment:
‘_MSC_VER‘ is a windows macro？

Reply:
I'm not sure, but if not modify this, it will compile error, so to avoid influence other platform, we add marco